src/database.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+from typing import Any
+
+from sqlalchemy import (
+    Boolean,
+    Column,
+    CursorResult,
+    DateTime,
+    ForeignKey,
+    Identity,
+    Insert,
+    Integer,
+    LargeBinary,
+    MetaData,
+    Select,
+    String,
+    Table,
+    Update,
+    func,
+)
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker
+
+from .config import settings
+
+# from constants import DB_NAMING_CONVENTION
+
+DATABASE_URL = str(settings.DATABASE_URL)
```

Comment:
Do we need to `str()` this instead of using this directly? I think it should already be a string